google-cloud-logging/lib/google/cloud/logging/logger.rb

```diff
@@ -54,10 +66,9 @@ class Logger
         attr_reader :labels
 
         ##
-        # A OrderedHash of Thread IDs to Stackdriver request trace ID. The
-        # Stackdriver trace ID is a shared request identifier across all
-        # Stackdriver services.
-        attr_reader :trace_ids
+        # A OrderedHash of Thread IDs to RequestInfo, keeping track of data
+        # associated with the request being handled by the current thread.
```

Comment:
I made `request_info` look up data for the current thread (and thus just return a `RequestInfo` rather than a hash), so it matches the behavior of `add_request_info` and `delete_request_info`.